CMakeLists.txt

```diff
@@ -399,6 +399,10 @@ endif()
 set_source(simdjson)
 resolve_dependency(simdjson 3.1.5)
 
+set_source(qatzstd)
+resolve_dependency(qatzstd)
```

Comment:
Is this supposed to be always added? I suspect this should be opt-in an behind an option? This will then also require the target to be added conditionally via an extra `target_link_libraries` in dwrf/common

Reply:
We initially designed it that way. But the PR is for evaluation and benchmarking, and for the convenience to switch qat on and off during benchmarking without re-build, we choose to include it at all times for now. The "production" version of the PR will use it as an option with default off.

Reply:
I see, makes sense!